packages/web/src/components/nav/desktop/NavPopupMenu.module.css

```diff
@@ -107,3 +107,8 @@
 .item:hover .usdcPill path {
   fill: #2775ca;
 }
+
+.item .audioPill path,
+.item:hover .audioPill path {
+  fill: revert-layer;
```

Comment:
cool! btw julian is considering having us use pngs for these given the complexity. other than the hover issue you just addressed, is there any other visual weirdness with using svgs for these?

Reply:
don't think so, the svg was not the issue really it was the popup menu

Reply:
the vast majority of places in the app were using the svgs before the migration